### PR TITLE
Add muted (disabled) k/wasm target in Compose libs

### PR DESCRIPTION
**Pull request by eymar**
This PR adds wasm target and source sets to some compose modules (not all for now. but if this change is accepted, I'll make this change for other modules too).
By default, it's disabled in gradle.properties - `kotlinWasmEnabled=false`

This change would help to sync wasm-main branch with jb-main branch. A lot of files in wasm-main branch were moved from jsMain source set to jsWasmMain (common source set between k/js and k/wasm). And now, when some changes happens in jb-main jsMain, it becomes less trivial to solve the conflict when merging it into wasm-main. 

This change mostly moves some files from jsMain to jsWasmMain. + adds some wasmMain files (disabled source set)

### CODE REVIEW

File: compose/runtime/runtime/src/wasmTest/kotlin/kotlinx/test/IgnoreTargetsActuals.wasm.kt [hunk 1]
```diff
@@ -14,19 +14,8 @@
...
-import org.khronos.webgl.get
-
-internal actual fun ByteArray.putBytesInto(array: IntArray, offset: Int, length: Int): Unit {
-   val byteBuffer = Uint8Array(this.toTypedArray()).buffer
-   val intArray = Int32Array(byteBuffer, offset, length)
-   for (i in 0 until intArray.length) {
-      array[i] = intArray[i]
-   }
-}
+package kotlinx.test
```
Comment (eymar) [inline compose/runtime/runtime/src/wasmTest/kotlin/kotlinx/test/IgnoreTargetsActuals.wasm.kt]: I can't understand why it considers this file as a moved one. It's a new file.

Comment (eymar) [reply compose/runtime/runtime/src/wasmTest/kotlin/kotlinx/test/IgnoreTargetsActuals.wasm.kt]: weird. IDEA thinks it's moved too

Comment (eymar) [reply compose/runtime/runtime/src/wasmTest/kotlin/kotlinx/test/IgnoreTargetsActuals.wasm.kt]: deleted the file and pushed it again. Seems to be fixed now.

Comment (Walingar) [reply compose/runtime/runtime/src/wasmTest/kotlin/kotlinx/test/IgnoreTargetsActuals.wasm.kt]: sometimes IDEA uses its own logic of detecting moves, so this is ok

**Review by Walingar — APPROVED** [review]
Cool, go ahead!

**Pull request merged**